src/pages/Product/Components/Detail/DetailBtn.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import CartModal from '../../../Cart/components/CartModal/CartModal';
+import './DetailBtn.scss';
+
+const DetailBtn = ({ count, price, id }) => {
+  const [isOpenCartModal, setIsOpenCartModal] = useState(false);
+  const navigate = useNavigate();
+
+  const randomNum = Math.floor(Math.random() * 300);
+
+  const addCart = () => {
+    const body = JSON.stringify({
+      user: '1',
+      product: id,
+      quantity: count,
+      price: price,
+    });
+
+    fetch('', {
```

Comment:
```suggestion
    // TODO: API 연결
    fetch('', {
```


이경우에는 주석등으로 마킹해두는게 좋아보이네요

Reply:
추후에 해야할 것을 찾기 쉽도록 TODO 주석을 달아놓았습니다.